api/v1alpha1/perconaservermysql_types.go

```diff
@@ -719,9 +736,19 @@ func (pmm *PMMSpec) HasSecret(secret *corev1.Secret) bool {
 }
 
 func (cr *PerconaServerMySQL) HAProxyEnabled() bool {
+	if !cr.Spec.AllowUnsafeConfig {
```

Comment:
Can we have `AllowUnsafeConfig=false` and `cr.Spec.Proxy.HAProxy.Enabled=false`?
The same applies for `OrchestratorEnabled()`.

Reply:
No, we can't